src/index.js

```diff
@@ -1,12 +1,38 @@
-const express = require("express")
+/* eslint-disable no-console */
+// TODO: Do we still need this.
+import 'source-map-support/register'
+
+// Do we still use regenerator?
+import "regenerator-runtime/runtime"
+
+// This must come before any other instrumented module.
+// See https://docs.datadoghq.com/tracing/languages/nodejs/ for more info.
+import "./lib/datadog"
+
+// TODO: Do we need JSDOM server-side?
+import "./lib/DOMParser"
```

Comment:
@eessex - I believe this was related to some publishing pop up thing that was only relevant on the client?

Reply:
I updated the comment when I discovered what it was used for.

`// Required for @artsy/react-html-parser which is included by @artsy/reaction`